### PR TITLE
MotionCompose: Update dependencies

### PR DESCRIPTION
**Pull request by yaraki**
Migrate from accompanist-insets to the official 'androidx.compose.foundation' insets.
https://google.github.io/accompanist/insets/#migration

Change-Id: I4a1593141c8232d94f2d2c995b03d327daa2669a

### CODE REVIEW

File: MotionCompose/app/build.gradle [hunk 1]
```diff
@@ -62,21 +62,20 @@ android {
...
+    implementation 'androidx.compose.material3:material3:1.0.0-alpha13'
     implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.4.0'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.4.1'
     implementation 'androidx.activity:activity-compose:1.4.0'
-    implementation 'androidx.constraintlayout:constraintlayout-compose:1.0.0-rc02'
+    implementation 'androidx.constraintlayout:constraintlayout-compose:1.0.1'
 
-    def accompanist_version = '0.22.0-rc'
+    def accompanist_version = '0.24.9-beta'
```
Comment (riggaroo) [inline MotionCompose/app/build.gradle:71]: I believe there is still this release blocking https://github.com/google/accompanist/pull/1186? @bentrengrove do we need to wait for accompanist to upgrade to beta03 of compose?

Comment (bentrengrove) [reply MotionCompose/app/build.gradle:71]: You should match your accompanist version to the compose version so technically yes

Comment (bentrengrove) [reply MotionCompose/app/build.gradle:71]: It probably will be fine though if you aren't using navigation-material

Comment (bentrengrove) [reply MotionCompose/app/build.gradle:71]: v0.24.10-beta is now out

Comment (yaraki) [reply MotionCompose/app/build.gradle]: Updated to v0.24.10-beta.

**Review by riggaroo — APPROVED** [review]

**Pull request merged**